heft-plugins/heft-serverless-stack-plugin/heft-plugin.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/heft-plugin.schema.json",
+
+  "taskPlugins": [
+    {
+      "pluginName": "ServerlessStackPlugin",
+      "entryPoint": "./lib/ServerlessStackPlugin",
+      "optionsSchema": "./lib/schemas/serverless-stack.schema.json",
```

Comment:
If there are no options, we shouldn't need to load a schema.

Reply:
I think not referencing a schema should signal to Heft that no options are allowed.